codes/quantum/qubits/quantum_expander.yml

```diff
@@ -0,0 +1,118 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: quantum_expander
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Quantum Expander code'
+
+# First reference with the code (optional).
+introduced: '\cite{doi:10.1109/FOCS.2015.55}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'CSS codes constructed from bipartite expander graphs with bounded left and right vertex degrees.'
```

Comment:
Can you expand on this a bit more by explaining how CSS code (or underlying classical code?) info corresponds to data from these graphs? Paper says something about how the parity check matrix is extracted from them.

Reply:
Updated the description.